src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ReturnCountCheck.java

```diff
@@ -137,6 +230,51 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public class MyClass {
+ *   public class FirstTest {
+ *     public FirstTest() {
+ *     } // OK
+ *   }
+ *
+ *   public class SecondTest {
+ *     public SecondTest() {
+ *       return;
+ *     } // OK
+ *   }
+ *
+ *   Operation firstOperation = (int x, int y) -&gt; x + y; // OK
+ *   Operation secondOperation = (int x, int y) -&gt; {
+ *     return x + y;
+ *   }; // OK
+ *   Operation badOperation = (int x, int y) -&gt; {
+ *     if (x &gt; y)
+ *       return x - y;
+ *     return x + y;
+ *   }; // violation, more than one return statements in lambda expressions
+ *
+ *   interface Operation {
```

Comment:
can you write a code that use some jdk standard lambda(without extra import is preferable, but ok to place import)  to avoid usage custom classes ?

Reply:
I didn't understand JDK standard lambda, Can you give me some resources to read about it?

Reply:
@Binita-tech in order to keep examples as simple as possible, we would rather not have extra classes/ interfaces included in examples unless necessary. In this case, you could use a `Predicate` to have similar syntax to show violations on:

```java
        final Predicate<Integer> p = i -> {
            if (i > 5) {
                return true;
            }
            return false;
        }; // violation

        final Predicate<Integer> p = i -> {
            return i > 5;
        }; // ok


```

Reply:
Done @nmancus1